server.js

```diff
@@ -79,6 +81,10 @@ function launchServer(options) {
       } else {
         app.use(express.static(path.join(__dirname, 'public')));
       }
+
+      app.use(bodyParser.json());
+      const api = API.install(app, gameServer);
```

Comment:
I don't know how lifetimes works. If I don't store the thing in a variable will it just get garbage collected? I need it to stay in memory as it keeps track of all of the HTTP connections.